packages/core/src/app/payment/storedInstrument/isInstrumentCardCodeRequired.ts

```diff
@@ -21,6 +21,11 @@ export default function isInstrumentCardCodeRequired({
         return !!paymentMethod.config.isVaultingCvvEnabled;
     }
 
+    // If the shipping address is untrusted, card verficiation mode has set with cvv, card code is required
+    if ((instrument as CardInstrument).untrustedShippingCardVerificationMode == 'cvv') {
```

Comment:
```suggestion
    if ((instrument as CardInstrument).untrustedShippingCardVerificationMode === 'cvv') {
```
Shouldn't we use an `enum` to hold the `cvv` and `pan` constants? Is it possible that we may need to add more mode types in the future?